cmake/modules/sysdig.cmake

```diff
@@ -22,7 +22,7 @@ endif()
 
 file(MAKE_DIRECTORY ${SYSDIG_CMAKE_WORKING_DIR})
 # cd /path/to/build && cmake /path/to/source
```

Comment:
I think that `SYSDIG_CHECKSUM` doesn’t get propagated here too now that we have it ?

Reply:
Actually that was on purpose, the use case mentioned was something like `cmake -DSYSDIG_VERSION=dev`

And then it was hard to provide a checksum (you need to download the tarball, even if you know the commit)

If you want to checksum also branches/commits provided by the user, it is possible to add another parameter.

For now, it will just download the file and continue with a `no URL_HASH specified?` warning, just like before...